.github/workflows/lint-test.yaml

```diff
@@ -50,3 +50,17 @@ jobs:
 
       - name: Run chart-testing (install)
         run: ct install --config ct.yaml
+
+      - id: get-version
+        name: Memorize pre-release version
+        run: |
+          echo version=$(helm show chart ./charts/flagsmith | grep -oP '^version: \K.*')-a$(date +"%Y%m%d%H%M%S") >> $GITHUB_OUTPUT
+
+      - name: Package the pre-release chart
+        run: helm package ./charts/flagsmith --version ${{ steps.get-version.outputs.version }} --destination out
+
+      - name: Save the pre-release chart archive
```

Comment:
What's the reasoning here?

Reply:
Trying to automate branch-based chart packaging so I can fiddle with Terraform with less hoops to jump through. Doesn't work as intended, I removed this.